test/views/stripe_customer_view_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule CodeCorps.StripeCustomerViewTest do
+  use CodeCorps.ConnCase, async: true
+
+  import Phoenix.View, only: [render: 3]
+
+  test "renders all attributes and relationships properly" do
+    user = insert(:user)
+    stripe_customer = insert(:stripe_customer, id_from_stripe: "some_id", email: "email", user: user)
+
+    rendered_json =  render(CodeCorps.StripeCustomerView, "show.json-api", data: stripe_customer)
+
+    expected_json = %{
+      "data" => %{
+        "attributes" => %{
+          "email" => stripe_customer.email,
+        },
```

Comment:
This appears to be missing attributes (and relationships) that are in the API Blueprint but not here.